src/plugins/workspace/server/routes/index.ts

```diff
@@ -91,19 +116,41 @@ export function registerRoutes({
       validate: {
         body: schema.object({
           attributes: workspaceAttributesSchema,
+          permissions: schema.maybe(
+            schema.oneOf([workspacePermission, schema.arrayOf(workspacePermission)])
+          ),
         }),
       },
     },
     router.handleLegacyErrors(async (context, req, res) => {
-      const { attributes } = req.body;
+      const { attributes, permissions: permissionsInRequest } = req.body;
```

Comment:
If user does not turn on `savedObjects.permission.enabled`, we should not save permissions into the workspace right?